software/control/src/CMakeLists.txt

```diff
@@ -123,7 +123,7 @@ set(java_src_files
   PoseGroundMonitor.java
   RobotPlanPublisher.java
   RobotStateCoder.java
-  AtlasStateExtraCoder.java
+#  AtlasStateExtraCoder.java
```

Comment:
I grep'ed the directory and didn't see usage. So  it can be

Funny because, I thought that the arm encoder calibration routine used a matlab instance to capture the end stops - that @mposa wrote.